packages/kernel-5.10/kernel-5.10.spec

```diff
@@ -96,9 +97,7 @@ scripts/kconfig/merge_config.sh \
   ../config-microcode \
 %endif
   %{SOURCE100} \
-%if %{_is_metal_variant}
-  %{SOURCE101}
-%endif
+  %{_sourcedir}/config-bottlerocket-%{_cross_variant_platform}
```

Comment:
For RPM packaging in general it's bad practice to conditionally include sources in a spec file, because then you might get a different SRPM based on the architecture of the Koji build host that builds it from the SCM checkout, or you might distribute an SRPM that only works for one of the distribution's supported architectures.

It's kind of irrelevant in the context of this project, where the source files are always available and SRPMs aren't even built, but I've worked with RPM packaging long enough to have an ingrained preference for doing it this way.